src/ucs/sys/topo.h

```diff
@@ -14,6 +14,8 @@
 
 BEGIN_C_DECLS
 
+#define UCS_TOPO_DEV_ID_UNKNOWN UINT_MAX /* indicates ucs_sys_bus_id_unknown */
```

Comment:
we should remove ucs_sys_bus_id_unknown altogether
if some transport wants to return unknown device - just return UCS_TOPO_DEV_ID_UNKNOWN
also, there is naming inconsistency:
 - ucs_sys_device_t vs. UCS_TOPO_DEV_ID_UNKNOWN
it should be one of:
 - ucs_sys_device_t and UCS_SYS_DEVICE_ID_UNKNOWN
 - ucs_topo_device_t vs. UCS_TOPO_DEVICE_ID_UNKNOWN

Reply:
This makes sense. Will use ucs_sys_device_t and UCS_SYS_DEVICE_ID_UNKNOWN

Reply:
> we should remove ucs_sys_bus_id_unknown altogether

This doesn't go well with memory attribute work that we have in progress. Current approach is 1. given a buffer return the bus_id associated with it 2. given a lane index (or some transport equivalent) return bus_id associated with it. We decided to add `ucs_sys_bus_id_unknown` in the first place because a) bus_id may not apply to NUMA nodes; b) transport may wish to return unknown bus id (may be by default if not supported). If we decide to remove this, then we need add api to allow transports to return device index UCS_SYS_DEVICE_ID_UNKNOWN directly.

Reply:
@yosefe any further thoughts on this? I don't see other dangling points on this PR besides this one.

Reply:
@Akshay-Venkatesh returning bus_id is NOT the approach. UCT lanes should return only sys_dev_id. bus_id is only between UCT and UCS.